src/Configuration.php

```diff
@@ -83,10 +86,22 @@ class Configuration implements ConfigurationInterface
      * @param CodeGeneratorListenerInterface[] $codeGeneratorListeners A list of listeners that can alter code generation of each bean/dao
      * @throws \Mouf\Database\SchemaAnalyzer\SchemaAnalyzerException
      */
-    public function __construct(string $beanNamespace, string $daoNamespace, Connection $connection, NamingStrategyInterface $namingStrategy = null, Cache $cache = null, SchemaAnalyzer $schemaAnalyzer = null, LoggerInterface $logger = null, array $generatorListeners = [], AnnotationParser $annotationParser = null, array $codeGeneratorListeners = [])
-    {
+    public function __construct(
+        string $beanNamespace,
+        string $daoNamespace,
+        string $resultIteratorNamespace,
```

Comment:
We need to take into consideration the impacts of adding a parameter here (rather than at the end of the constructor).
This will introduce breaking changes (in Laravel / Symfony / Mouf packages) and we must be sure to handle them correctly.

Reply:
`$resultIteratorNamespace` is now moved at the end of the contructor and defaults to `Dao\..\ResultIterator` namespace